docs/src/index.md

```diff
@@ -232,21 +246,25 @@ correctly and thus returns fundamentally wrong results.
 
 At the moment there is limited support for sparse linear algebra operations. Sparse arrays may be used, but care must be taken because backing arrays drop zeros in Julia (unless told not to).
 
-```julia
+```jldoctest sparse
 using SparseArrays
+a = sparse([2.0])
+da1 = sparse([0.0]) # Incorrect: SparseMatrixCSC drops explicit zeros
+Enzyme.autodiff(Reverse, sum, Active, Duplicated(a, da1))
+da1
 
-a=sparse([2.0])
-f(a)=sum(a)
+# output
+
+1-element SparseVector{Float64, Int64} with 0 stored entries
+```
 
-# Incorrect: SparseMatrixCSC drops explicit zeros
-# returns 1-element SparseVector{Float64, Int64} with 0 stored entries
-da=sparse([0.0])
+```jldoctest sparse
+da2 = Enzyme.make_zero(a) # Correct: Prevent SparseMatrixCSC from dropping zeros
```

Comment:
I wouldn't replace the explicit call to `da=sparsevec([1], [0.0])` here.

Instead, I would add a section at the end that says something like "Since determining how to perform this zeroing can be complicated, Enzyme provides a helper function `Enzyme.make_zero` that does this automatically.